test/functionalTest/cases/2507_csub_metadata_field/special_metadata_in_notifications_ngsiv1.test

```diff
@@ -97,7 +97,7 @@ payload='{
     },
     "attrs": [ "A", "B", "C" ],
     "attrsFormat": "legacy",
-    "metadata": [ "previousValue", "actionType" ]
```

Comment:
With the changes in the NGSIv1 renders (WI4) now the order of the attributes in metadata matters also for NGSIv1. I see this as a kind of improvement.

Reply:
NTC